packages/e2e-tests/src/specs/editor/taxonomy.js

```diff
@@ -49,41 +49,56 @@ async function goToAndExpandTaxonomyPanel() {
 
 async function addCategory(name, parent) {
   await expect(page).toClick('button', { text: 'Add New Category' });
-  await expect(page).toFill('input[name="New Category Name"]', name);
 
   if (parent) {
     await expect(page).toClick('button[aria-label="Parent Category"]');
     await expect(page).toMatchElement('li[role="option"]', { text: parent });
-    await expect(page).toClick('li[role="option"]', { text: parent });
-    await expect(page).toMatchElement(
-      'li[role="option"][aria-selected="true"]',
-      { text: parent }
-    );
 
+    await expect(page).toClick('li[role="option"]', { text: parent });
     await expect(page).toMatchElement('button[aria-label="Parent Category"]', {
       text: parent,
     });
-
-    // TODO: Add assertion here to verify the parent was chosen and the dropdown closed.
   }
 
+  await expect(page).toFill('input[name="New Category Name"]', name);
   await page.keyboard.press('Enter');
-
   await expect(page).toMatchElement('label', {
     text: name,
   });
 }
 
+async function addTag(name) {
+  await page.focus('input#web_story_tag-input');
+  await page.type('input#web_story_tag-input', name);
+  await page.keyboard.press('Enter');
+
+  await page.focus('input#web_story_tag-input');
+
+  const tokens = await page.evaluate(() =>
+    Array.from(
+      document.querySelectorAll('[data-testid="flat-term-token"] > span'),
```

Comment:
Since the tags are in spans there wasn't really a great way to zero in on them besides this? Down for alternatives, but this lets us check that the term asked for has made it to the list.